repos/system_upgrade/el7toel8/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -4,13 +4,21 @@
 from leapp import reporting
 from leapp.exceptions import StopActorExecution, StopActorExecutionError
 from leapp.libraries.actor import constants
-from leapp.libraries.common import dnfplugin, mounting, overlaygen, repofileutils, rhsm, utils
+from leapp.libraries.common import (
```

Comment:
two different import styles in the same file, it would be great to have them at least the same formatting in the same file. Thanks.

Reply:
sure, makes sense...